### PR TITLE
opn@6.0.0: The package has been renamed to `open`

### PR DESCRIPTION
**Pull request by xmtigra**
npm WARN deprecated opn@6.0.0: The package has been renamed to `open`

### CODE REVIEW

**Comment by alan-agius4** [comment]
Hi, try to run;

```
git reset --soft HEAD~
git commit --all -m "refactor: replace deprecated package opn with open"
git push --force
```

**Comment by alan-agius4** [comment]
You still have two commits :P

**Comment by xmtigra** [comment]
I've fixed it. Thank you for your help

**Comment by alan-agius4** [comment]
You are welcome 😊

File: tests/legacy-cli/e2e/tests/build/styles/material-import.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export default function () {
 
   const extensions = ['css', 'scss', 'less', 'styl'];
   let promise: Promise<any> = Promise.resolve()
-    .then(() => silentNpm('install', '@angular/material@5.0.4'));
+    .then(() => silentNpm('install', '@angular/material@7.3.6'))
```
Comment (alan-agius4) [inline tests/legacy-cli/e2e/tests/build/styles/material-import.ts]: Actually can you revert the change in this file?

Comment (xmtigra) [reply tests/legacy-cli/e2e/tests/build/styles/material-import.ts]: done
